File: 8kyu/(8 kyu) If you can't sleep, just count sheep/(8 kyu) If you can't sleep, just count sheep.cs
using System;

public static class Kata
{
  public static string CountSheep(int n)
  {
    string res = "";
    
    for (int i = 1; i <= n; i++)
    {
      res += $"{i} sheep...";
    }
    
    return res;
  }
}

File: 8kyu/(8 kyu) If you can't sleep, just count sheep/(8 kyu) If you can't sleep, just count sheep.rs
fn countSheep(n: u32) -> String {
    // #1
    // let mut result = String::new();
    // for i in 0..n {
    //     let x = format!("{} sheep...", i + 1);
    //     result.push_str(&x);
    // }
    // return result;
    // #2
    let mut result = String::new();
    for i in 0..n {
        result.push_str(&format!("{} sheep...", i + 1));
    }
    result
}
